cluster/gce/upgrade.sh

```diff
@@ -447,6 +447,27 @@ if [[ "${master_upgrade}" == "false" ]] && [[ "${node_upgrade}" == "false" ]]; t
   exit 1
 fi
 
+# prompt if etcd storage media type isn't set
+if [[ -z "${STORAGE_MEDIA_TYPE:-}" ]]; then
```

Comment:
My only concern with that is then `prepare-upgrade` gets called before the prompt. This includes uploading tars to GCS which can take some time and could mean someone could walk away assuming there upgrade was completing when it was sitting on the prompt. WDYT?